private/bufpkg/bufmodule/module.go

```diff
@@ -61,11 +61,16 @@ func newModuleForProto(
 	if err != nil {
 		return nil, err
 	}
-	var breakingConfig *bufbreakingconfig.Config
+	// If there is no breaking and/or lint config, we want to default to the v1beta1 version
+	breakingConfig := &bufbreakingconfig.Config{
+		Version: bufconfig.V1Beta1Version,
+	}
 	if protoModule.GetBreakingConfig() != nil {
 		breakingConfig = bufbreakingconfig.ConfigForProto(protoModule.GetBreakingConfig())
 	}
-	var lintConfig *buflintconfig.Config
+	lintConfig := &buflintconfig.Config{
+		Version: bufconfig.V1Beta1Version,
+	}
```

Comment:
I think we still need to verify that those versions are always the same right? Imagine the case when a `v1` lint configuration was found, but we default to `v1beta1` for the breaking configuration.

Thinking through our options, it might be best to attempt to default to `v1beta1` if the configuration is not present, but return an error if lint and breaking are mismatched. So if the Protobuf message contains a `v1` version for one of the configurations but not the other, we would have an error.

What are your thoughts?

Reply:
Oh hmm, that is... probably reasonable, tbh. We _shouldn't_ hit the case where if neither is `nil`, they're mismatched, but that's probably good to check here as a safety thing, similar to what we do in `WriteConfig`.